configuration/openmrs/i18n/clinical/locale_fr.json

```diff
@@ -0,0 +1,402 @@
+{
+  "MODULE_LABEL_ACTIVE_KEY": "Actifs",
+  "MODULE_LABEL_PROGRAMS_KEY": "Programmes",
+  "MODULE_LABEL_MY_PATIENTS_KEY": "Mes Patients",
+  "MODULE_LABEL_ACTIVE_PATIENTS_BY_LOCATION": "Patients/Emplacement",
+  "MODULE_LABEL_OPD_1_KEY": "OPD-1",
+  "MODULE_LABEL_ALL_KEY": "Tous",
+  "MODULE_LABEL_EMERGENCY_KEY": "Urgents",
+  "MODULE_LABEL_PRIORITY_KEY": "Prioritaires",
+  "MODULE_LABEL_QUEUE_KEY": "Non Urgents",
+  "MODULE_LABEL_NO_VITALS_KEY": "Signes vitaux non pris",
```

Comment:
Label as "En attente des signes vitaux" (if that's not too long).

Reply:
[Done](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/7/commits/318c6c514390c51e164fd68b568bb984926f73b2)